CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 21.2.0-RC1
 
 ### 21.2 Breaking Changes
-There are currently no breaking changes in 21.2
+`--skip-pow-validation-enabled` is now an error with `block import --format JSON`. This is because the JSON format doesn't include the nonce so the proof of work must be calculated.
```

Comment:
```suggestion
* `--skip-pow-validation-enabled` is now an error with `block import --format JSON`. This is because the JSON format doesn't include the nonce so the proof of work must be calculated.
```